SITCOMTN-023.tex

```diff
@@ -49,13 +49,158 @@
 
 % ADD CONTENT HERE
 % You can also use the \input command to include several content files.
+\section{Introduction}
+The goal of this document is to define the process for an efficient and timely organization of the work required for the successful commissioning of the Vera C. Rubin Observatory. 
+This process includes defining the current SIT-Com priorities along with their coordination both at the SIT-Com and Project level. 
+The global prioritization of activities is defined by three primary use-cases: (i) the level 2 and level 3 milestones described in P6, (ii) planned work not linked to a milesone (including the managerial work from the SIT-Com leadership team (SCLT)), and (iii)  unplanned emergent issues.
+
+This document first lays down the different use cases along with their appropriate processes and the prioritization strategy.
+ From these uses cases, we defined the requirements on the Jira tool that will satisfy our needs, including work flow, ticket types, review process, dashboard. 
+ Finally, we describe how this Jira project interfaces with the other existing projects. 
+
+\section{Work Management Needs}
+\subsection{Processes}
+As mentioned in the introduction, there are three main use cases for the work management process. 
+\subsubsection{Milestone Tracking}
+The P6 tool defined two sets of milestones for SITCom, Level 2 (L2) and Level 3 (L3). Each milestone is reached when the list of associated activities is completed. 
+Both L2 and L3 milestones will be reported on weekly at the SITCom leadership meetings, especially if delays/blockers are encountered, in which case the impact on the schedule/cost/performance might need to be elevated.
+
+\subsubsection{Planned activities not linked to a milestone}
+The tickets types and processes are similar to the milestone tracking use case, except that it would start at the Epics level or the Story level, depending on the nature of the task. 
+
+\subsubsection{Emergent Tasks}
+Finally, the emergent tasks use case is a mechanism for anyone on the commissioning team to enter issues and bugs encountered during commissioning activities. 
+They do not need to be linked to an epic or a milestone while being created. 
+The triage will be done by the SCLT to verify and/or define assignees as well as priorities relative to other work. During this process they will either be added to an active epic or put on the backlog for later scheduling.
+
+\subsection{Prioritization strategy}
+In order to assign the priority accordingly, the activities will need to have a criticality attached to them.  
```

Comment:
"Criticality" seems weird. Jira calls them a priority, so we should do something similar.
Consider:
Activities are assigned a priority level to help guide their order of implementation.
The priority level is first assigned by the reporter of the ticket and then discussed and updated by the SCLT if it is appropriate to do so.
The 5 priority levels are defined as follows:

Reply:
Typically a severity and priority are assigned.  Severity decribes the degree of impact an issue has on the system nder test, e.g 'total system failure' or just 'annoying but does not cause harm' . The submitter assignes a severity when submitting the bug. Priority is order in which bugs should be fixed. A severe impact bug may only happen  rarely so may not be a priority to fix. priority is asssigned by the CCB when deciding what order in which to do work.  

I think criticality here is meant to reflect severiry, not priority?

Reply:
The criticality was priority so I changed to the recommended text Patrick wrote. Now I do like your suggestion of the severity... which would be very handy with the missing capability. Could you write a ticket for this one and assign it to Austin (with me as a watcher). It is not implemented right now.

Reply:
GH isn't showing the changed text... which is annoying.
I consider this resolved but will leave Leanne to press the button.